src/main/java/dev/hephaestus/atmosfera/AtmosferaConfig.java

```diff
@@ -196,14 +198,14 @@ public static Screen getScreen(Screen parent) {
 					// Replaces the "colon" with a "dot" as the ID separator to utilize the language file.
 					String soundLangID = String.join(".", sound.getKey().toString().split(":"));
 
-					TranslatableText subtitleText = new TranslatableText("subtitle." + soundLangID);
-					LiteralText tooltipText = new LiteralText(soundLangID + "\n" + I18n.translate(subtitleText.getKey()));
+					MutableText tooltipText = Text.literal(soundLangID + "\n");
+					tooltipText.append(Text.translatable("subtitle." + soundLangID));
```

Comment:
Instead of "live" translating and combining the strings, simply append a translatable text.